lisp/ess-julia.el

```diff
@@ -281,19 +71,19 @@ VISIBLY is not currently used."
       (kill-buffer)
       (nreverse out))))
 
-(defvar julia--manual-topics nil)
-(defun julia-manual-lookup-function (&rest args) ; args are not used
+(defvar ess-julia--manual-topics nil)
+(defun ess-julia-manual-lookup-function (&rest args) ; args are not used
   (interactive)
   "Look up topics at http://docs.julialang.org/en/latest/manual/"
   ;; <li class="toctree-l1"><a class="reference internal" href="introduction/">Introduction</a></li>
-  (let* ((pages (or julia--manual-topics
-                    (setq julia--manual-topics
-                          (julia--retrive-topics "http://docs.julialang.org/en/latest/manual/"))))
+  (let* ((pages (or ess-julia--manual-topics
+                    (setq ess-julia--manual-topics
+                          (ess-julia--retrive-topics "http://docs.julialang.org/en/latest/manual/"))))
```

Comment:
I didn't add this. I only renamed the function. This functionality is already part of ESS: https://github.com/emacs-ess/ESS/blob/e16e13d36a1e3fce928d1434f07e0dfd96cdf218/lisp/ess-julia.el#L294

If you'd like to remove it, it's your code :). I don't think it's relevant to this pull request though.